nuxt.config.js

```diff
@@ -9,5 +9,8 @@ export default theme({
       name: 'English'
     }],
     defaultLocale: 'en'
+  },
+  build: {
+    publicPath: 'https://docs.sigstore.dev'
   }
-})
+})
```

Comment:
nit: new line

Reply:
Thanks @cpanato I added the line did not mean to dismiss the review